client/src/api/socketService.js

```diff
@@ -22,9 +22,20 @@ const leaveGame = () => {
   socket.emit('disconnect');
 };
 
+const updateGameState = ({ pacman, roomId, playerId }) => {
+  socket.emit('update-game-state',
```

Comment:
Also this is not the right way. This will emit the data to all the clients connected to the server and we want to send to a particular room.

Reply:
Yeah, I agree. This is done for simplicity. Right now, we can try to build for just 2 players & 1 room. When it's done then, can send it just to the users in that room.
Also creating the playerId on the client is a bad idea. It should be done on the server.